build.sbt

```diff
@@ -8,14 +8,20 @@ resolvers += "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases"
       
 scalaVersion := "2.11.11"
 
-
 libraryDependencies ++= Seq( jdbc , cache , ws , specs2 % Test,
```

Comment:
Can you clean up the dependencies and have them all added through the `Seq` doesn't look gr8 all over the place